src/gravel/api/auth.py

```diff
@@ -0,0 +1,56 @@
+# project aquarium's backend
+# Copyright (C) 2021 SUSE, LLC.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+from logging import Logger
+from fastapi.logger import logger as fastapi_logger
+from fastapi.routing import APIRouter
+from fastapi import HTTPException, Request, Response, Depends, status
+from fastapi.security import OAuth2PasswordRequestForm
+from pydantic import BaseModel, Field
+from gravel.controllers.auth import jwt_auth_scheme, JWTDenyList, JWTManager, JWT, UserMgr
+
+
+logger: Logger = fastapi_logger
+
+router: APIRouter = APIRouter(
+    prefix="/auth",
+    tags=["auth"]
+)
+
+
+class LoginReplyModel(BaseModel):
+    access_token: str = Field(title="The access token")
+    token_type: str = Field("bearer")
+
+
+@router.post("/login", response_model=LoginReplyModel)
+async def login(request: Request, response: Response,
+                form_data: OAuth2PasswordRequestForm = Depends()) -> LoginReplyModel:
```

Comment:
I'd prefer the return to be moved to the next line and properly limit the length at 80 cols. Something like

```
async def login(request: Request, response: Response,
                form_data: OAuth2PasswordRequestForm = Depends()
) -> LoginReplyModel:
```

Reply:
> I'd prefer the return to be moved to the next line and properly limit the length at 80 cols. Something like
> 
> ```
> async def login(request: Request, response: Response,
>                 form_data: OAuth2PasswordRequestForm = Depends()
> ) -> LoginReplyModel:
> ```

My IDE does not like that and shows a PEP8 error. Instead we should format it this way:
```
async def login(
        request: Request, response: Response,
        form_data: OAuth2PasswordRequestForm = Depends()
) -> LoginReplyModel:
```

Reply:
I most definitely agree. I just didn't want to go and be too annoying :)